src/tests/multihost/alltests/conftest.py

```diff
@@ -7,6 +7,7 @@
 import ldap
 import pytest
 import re
+import subprocess
```

Comment:
Why is this needed?

Reply:
This file is using the subprocess module. As an [example](https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/conftest.py#L246). The question would be, why was it working previously? I guess one of the modules imported by `conftest.py` already imported `subprocess` and when I removed it then the test started failing.